rosdep/python.yaml

```diff
@@ -6420,3 +6420,6 @@ yapf:
     yakkety:
       pip:
         packages: [yapf]
+yapf3: 
+  ubuntu: 
+    focal: [yapf3]
```

Comment:
Since this is available on other distributions than focal, we should add support for them.  The most forwards-compatible way here would be this:

```
yapf3:
  debian: [yapf3]
  fedora: [python3-yapf]
  ubuntu:
    '*': [yapf3]
    xenial: null
```